### ISSUE TITLE
Compiling on Win32

### ISSUE THREAD

**Issue by schmod** [chobie/php-uv#25]
What are the instructions for compiling on win32?  Unless I'm mistaken, phpize is not available on that platform.

**Comment by chobie** [comment]
sorry for late response. you can build php-uv like this.
also, you need to adjust php-sdk versions and PHP's configure options (thread safe, debug mode etc...)

```
# open Windows SDK *** Command Prompt
cd path\to\php-sdk
bin\phpsdk_setvars.bat
cd path\to\php-uv
# you need to build libuv at first (just type vcbuild.bat)
phpize
configure --enable-uv
nmake
```

but. currently it can't build on windows as there are some compile errors :sweat:
I'll fix this problem in this week.

**Comment by chobie** [comment]
fixed in a62a234d497ff3fbfa2ef137a225ee30aae6eda8. you also need to specify `--enable-sockets` flag when building php.
